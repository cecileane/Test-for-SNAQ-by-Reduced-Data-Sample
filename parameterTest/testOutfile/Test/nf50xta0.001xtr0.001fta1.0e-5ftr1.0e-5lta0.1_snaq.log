optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 50, liktolAbs = 0.1.
rootname for files: nf50xta0.001xtr0.001fta1.0e-5ftr1.0e-5lta0.1_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 31549
seed: 31549 for run 1
Sat Oct  1 17:30:07 2016

 BEGIN SNaQ for run 1, seed 31549 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 98 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,10.0,0.48942,1.51213,0.51916]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 28		 28		 2	 |	 0		 0		 --
mvtarget	 21		 21		 2	 |	 0		 0		 --
chdir		 12		 12		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 36		 28		 0	 |	 0		 0		 --
Total		 98		 90		 5	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981975001
((3,(5,(6,#H7:::0.2856030478731591):10.0):0.5191607665951044):0.4894188885107836,4,(2,(1)#H7:::0.714396952126841):1.512133479379108);
---------------------
seed: 38173 for run 2
Sat Oct  1 17:30:13 2016

 BEGIN SNaQ for run 2, seed 38173 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 83 (failures: 50) with -loglik=155.49626 and ht_min=[0.33535,0.12573,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 27		 27		 1	 |	 2		 2		 --
mvtarget	 17		 17		 1	 |	 2		 2		 --
chdir		 6		 6		 1	 |	 4		 4		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 31		 22		 1	 |	 1		 1		 --
Total		 83		 74		 5	 |	 9		 9		 5
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 2, -loglik of best 155.4962573455153
(1,(((5,6):0.3353472615210538,3):0.1257331528330909,(4,#H7:::0.37302473529250035):2.155859703631241):2.155859703631241,(2)#H7:::0.6269752647074995);
---------------------

Sat Oct  1 17:30:18 2016
MaxNet is ((3,(5,(6,#H7:::0.2856030478731591):10.0):0.5191607665951044):0.4894188885107836,4,(2,(1)#H7:::0.714396952126841):1.512133479379108); 
with -loglik 126.74250981975001
