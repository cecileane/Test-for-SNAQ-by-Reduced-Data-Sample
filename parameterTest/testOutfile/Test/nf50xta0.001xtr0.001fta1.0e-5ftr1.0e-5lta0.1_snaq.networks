((3,(5,(6,#H7:::0.2856030478731591):10.0):0.5191607665951044):0.4894188885107836,4,(2,(1)#H7:::0.714396952126841):1.512133479379108);, with -loglik 126.74250981975001 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((3,(5,(6)#H7:::0.6247853342727325):9.051790786118733):0.46090955415869017,4,(2,(#H7:::0.37521466572726747,1):1.2593070944621996):0.6604235426710927);, with -loglik 130.60666890295423
((3,(5,(6,((2)#H7:::0.5529593256573857,1):4.0013710551480575):0.13849527884389354):0.08284090739731158):1.079528620662014,4,#H7:::0.4470406743426143);, with -loglik 198.2515504750478
(3,(4)#H7:::0.5065730118629777,(5,(6,((#H7:::0.4934269881370224,2):0.24332425513597075,1):1.0978297091338018):0.22280332692253202):0.11421888041559299);, with -loglik 201.48563991010082
((3)#H7:::0.5988995580674511,4,(2,(((5,#H7:::0.4011004419325489):9.527571462312677,6):0.9933424747263051,1):0.08180501059208324):0.07551977923391906);, with -loglik 215.17035187617202
((3,(5)#H7:::0.5043034917160643):1.1493745866291638,4,(2,((#H7:::0.49569650828393574,6):5.678868744175467,1):0.02983405495093419):0.21397820518748975);, with -loglik 231.217352191585
