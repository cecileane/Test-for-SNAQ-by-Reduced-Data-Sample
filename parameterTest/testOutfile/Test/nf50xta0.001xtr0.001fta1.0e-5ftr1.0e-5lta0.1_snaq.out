((3,(5,(6,#H7:::0.2856030478731591):10.0):0.5191607665951044):0.4894188885107836,4,(2,(1)#H7:::0.714396952126841):1.512133479379108); -Ploglik = 126.74250981975001
 Dendroscope: ((3,(5,(6,#H7):10.0):0.5191607665951044):0.4894188885107836,4,(2,(1)#H7):1.512133479379108);
 Elapsed time: 11.11780332 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((3,(5,(6,#H7:::0.2856030478731591):10.0):0.5191607665951044):0.4894188885107836,4,(2,(1)#H7:::0.714396952126841):1.512133479379108);, with -loglik 126.74250981975001
 (1,(((5,6):0.3353472615210538,3):0.1257331528330909,(4,#H7:::0.37302473529250035):2.155859703631241):2.155859703631241,(2)#H7:::0.6269752647074995);, with -loglik 155.4962573455153
-------
